### DISCUSSION TITLE
Can I hide the spinner and blank overlay on the Sync login screen to sucessfully log in on FF68 mobile?

### DISCUSSION THREAD

**Discussion by mzso** [Aris-t2/CustomCSSforFx#589] [Q&A]
Hi!

I have this older instance running. And I want to sync its content with newer editions. But the login screen is broken, infinite spinner on a blank white background.
By chance I found that the login form is invisibly there. And I can successfully hit elements by tapping blindly. But never managed to successfully fill it.

Is there a way to apply some CSS to the about:accounts login page, to hide the spinner and the overlay?

**Comment by Aris-t2** [comment]
I don't think you can apply custom CSS to Firefox mobile.

Temporarily rename /chrome/ folder in your profile directory, if custom code causes issues with your older desktop setup of Firefox.

You can also transfer you "old" profile or a copy of it to a new setup. I suggest to use "Portable Firefox" for this and just copy and paste the content of the old setup into the new one.

**Reply by mzso** [reply]
Oh well, it's not possible to access the profile folder either on Android.

**Reply by Aris-t2** [reply]
The profile folder answer was for the desktop version. 
Why not just updating to a recent Firefox version on Android? You can get a version working on Android 5+ manually too, e.g. from here (Adblocker recommended): https://www.apkmirror.com/apk/mozilla/firefox/firefox-112-0-release/

**Reply by mzso** [reply]
I see. Well, I want to keep this instance as it is, but I also want to get the data out of it. But I'm running out of ideas...
